src/bot.js

```diff
@@ -12,13 +12,18 @@ const config = JSON.parse(
 bot.config = config;
 bot.commands = new Commands(bot);
 bot.logger = new Logger();
-bot.blacklist = new Discord.Collection();
+bot.blacklist = fs.readFileSync("./blacklist.txt","UTF-8").split('\n');
 
 const events = fs.readdirSync(path.join(__dirname, 'events'));
 for (const event of events) {
     const name = event.split('.')[0];
     const eventFunc = require(path.join(__dirname, 'events', name));
     bot.on(name, (...args) => eventFunc.run(bot, ...args));
 }
-
+bot.on('message',message=>{
+    if(bot.blacklist.some(a=>message.content.toLowerCase().includes(a.toLowerCase()))){
+        message.delete(500);//In case this is triggered before the message is actually sent, so that way there is no undeletable messages (I have experience with this problem)
+        message.channel.send(message.author.toString()+" Hi! Please don't make use of obscene/ discriminatory words, thanks!")
+    }
+})
```

Comment:
Please add this to the command manager in the `handleCommand()` function. You should take a look into the structure of the bot before adding something :wink:

Reply:
Also for the problem you mentioned, that can be solved by using async await